src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -2660,5 +2630,89 @@ public object ConvertBack(object value, Type targetType, object parameter, Syste
                 throw new Exception("The method or operation is not implemented.");
             }
 
-        }               
-   }
+        }
+
+        public class ClassViewMarginConverter : IValueConverter
+        {
+            private const int levelmargin = 45;
+            private const int marginTop = -10;
+            private const int marginLeft = -10;
+            private const int viewMarginLeft = -30;
+            private const int factor = 10;
```

Comment:
http://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants